composer/algorithms/stochastic_depth/stochastic_depth.py

```diff
@@ -130,9 +131,9 @@ class StochasticDepth(Algorithm):
             ``"uniform"`` assigns the same ``drop_rate`` across all layers.
             ``"linear"`` linearly increases the drop rate across layer depth
             starting with 0 drop rate and ending with ``drop_rate``.
-        drop_warmup: Percentage of training epochs to linearly
-            increase the drop probability to `linear_drop_rate`. Must be between
-            0.0 and 1.0.
+        drop_warmup (str | Time | float): A :class:`Time` object, time-string, or float
```

Comment:
Since you're editing this can you also add typing to the other args here?

Reply:
https://github.com/mosaicml/composer/pull/602 added those in.